apps/browser/src/backend/services/toolbox/services/shell/integrations/bash.sh

```
@@ -0,0 +1,56 @@
+  printf '\033]133;B\007'
+  printf '\033]133;C\007'
+}
+
+# Install hooks
+trap '__stagewise_pre_exec' DEBUG
+
+# Append to PROMPT_COMMAND (support both string and array forms in bash 5.1+)
+if [[ ${BASH_VERSINFO[0]} -ge 5 && ${BASH_VERSINFO[1]} -ge 1 ]]; then
+  PROMPT_COMMAND+=('__stagewise_prompt_command')
+else
+  if [[ -z "$PROMPT_COMMAND" ]]; then
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)